packages/Text/src/styles.ts

```diff
@@ -39,6 +60,9 @@ export const Text = styled.p<TextOptions>(({ lines, variant }) => {
       ${system};
     }
 
-    ${system};
+    ${lines &&
+    lines !== Infinity &&
+    Object.keys(lineHeightHeadingsFixer).includes(variant) &&
+    fixLineHeightStyles(variant)};
```

Comment:
i dont understand how this is related to the upgrade ?

Reply:
It has nothing to do with the upgrade indeed 🙈
Just couldn't keep up with those g, y, q, j and p being cut